lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -34,6 +34,16 @@ export class BrowserUtils {
         window.location.hash = "";
     }
 
+    /**
+     * Replaces current hash with hash from provided url
+     */
+    static replaceHash(url: string): void {
+        const urlParts = url.split("#");
```

Comment:
You can use `UrlString.removeHash()` here

Reply:
I'm trying to retrieve the right side of the split here. `UrlString.removeHashFromUrl()` returns the left side

Reply:
nvm, looks fine.